docs/GraFx-Studio/guides/template-variables/define/index.md

```diff
@@ -59,14 +59,32 @@ This location does not have any effect on the working of your document. It will
 
 ## Types
 
+Create a variable. Then, in the properties panel, choose the variable type. Different use cases demand different variable types.
+
+![screenshot](vartypes.png)
+
 ### Single Line of Text
 
 A placeholder for text.
 
+[Use Text variables](/GraFx-Studio/guides/template-variables/define/)
```

Comment:
This link refers to the same page.

Reply:
Will remove the link (for now), page does not exist yet. Will re-add later